lab_1_classify_by_unigrams/main.py

```diff
@@ -98,10 +214,23 @@ def detect_language_advanced(unknown_profile: dict[str, str | dict[str, float]],
     :param known_profiles: a list of known profiles
     :return: a sorted list of tuples containing a language and a distance
     """
+    if not isinstance(unknown_profile, dict) or not isinstance(known_profiles, list):
+        return None
+
+    detection = []
+    for profile in known_profiles:
+        result_of_comparison = compare_profiles(unknown_profile, profile)
+        detection.append((profile['name'], result_of_comparison))
+
+    detection.sort(key=lambda a: (a[1], a[0]))
+    return detection
 
 
 def print_report(detections: list[tuple[str, float]]) -> None:
     """
     Prints report for detection of language
     :param detections: a list with distances for each available language
     """
+    for language in detections:
+        print(f'{language[0]}: MSE {language[1]:.5f}')
+   
```

Comment:
strange change

Reply:
What exactly does look strange? As I see I get a list with a tuple and then I print only name of a language and an appropriate MSE score. I can’t understand properly: do I lose something?